libs/SmartSync/SmartSync.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 49;
```

Comment:
I don't know where this came from but 48 is the latest.  This being 49 breaks some CI tools.